spec/models/booking_spec.rb

```diff
@@ -9,4 +9,17 @@
     it { should validate_presence_of(:mobile).with_message("Mobile Should not be empty") }
 
   end
+
+  context 'scope tests' do
+    booking = FactoryBot.create(:booking)
+
+    it 'should return booked meeting rooms count' do
+      expect(Booking.booked.size).to eq(3)
```

Comment:
Booking is created only once using FactoryBot then why there are three booked meeting rooms?

Reply:
Changes done.